routes/web.php

```diff
@@ -14,15 +15,22 @@
 | contains the "web" middleware group. Now create something great!
 |
 */
+
+Route::controller(ShowController::class)->group(function() {
+    Route::get('/index', 'index')->name('index');
+    Route::get('/index/article/{id}', 'show')->name('article.show');
+});
+
+
 Auth::routes();
 Route::group(['middleware'=> 'auth'],function(){
     Route::controller(ArticleController::class)->group(function() {
```

Comment:
```suggestion
    Route::controller(ArticleController::class)->group(function() {
```

これ後でこの１つのコントローラしか使えないの足枷になりそうだから、管理画面全体を1コントローラでまとめるのやめた方がいいと思う。

記事、カテゴリ、設定とかなんでも詰め込むの辛いわ多分